cmd/root.go

```diff
@@ -463,6 +463,12 @@ func secrets() []string {
 		debugFlags,
 		viper.GetString(config.DEBUG_FLAGS_KEY),
 	))
+	// append log level from configuration
+	logLevel := os.Getenv(rootCtxt.appCtx.LogLevelEnvVar())
```

Comment:
I think we should get the log level from the configuration, and expose it to the sub command